### DISCUSSION TITLE
Evaluate the usage of fvm

### DISCUSSION THREAD

**Discussion by JulianBissekkou** [leancodepl/patrol#1222] [General]
Many peope use [fvm](https://fvm.app/)  to manage all the Flutter sdks.
Patrol currently requires a flutter installation as path which is not the case when using fvm.
It would be nice to be able to configure patrol to point to a specific sdk.

We could easily link patrol to a installed flutter version.
I am curious if anybody had similar issues :)

**Comment by bartekpacia** [comment]
Hi 👋🏻 

I also use `fvm`, but I always do (for example) `fvm global 3.7.12` so I can use the `flutter` command. I don't like having to prepend every invocation with `fvm` :)

I understand that your request is to make Patrol work when `fvm flutter` is available, but `flutter` is not?

**Reply by JulianBissekkou** [reply]
I don't want a hard link between patrol and fvm. Let me explain what I mean by an example.
Let's say you have a project setup and your fvm config in that project configures the flutter version 3.7.11. Then fvm will create a link in the project root to the correct sdk.
When I now run the patrol_cli I want to make sure that the flutter version **of that project** (available in `.fvm/` folder) is used.

My solution was to simply export the flutter sdk stored in `.fvm/` from the project before invoking patrol.
This should work... 🤔 
Well I think this disucssion was enough help to figure things out 😆 

I don't see a reason for patrol to explicitly add support for this case, what do you think?

**Reply by bartekpacia** [reply]
Thanks for explaining, and I'm happy you found a solution.

I'd also prefer not to implement this. Patrol only expects the `flutter` command to be available – how you make it available (global flutter install, `fvm`, or something else) is entirely your thing and we just don't care :)
